ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/QueryEndpoint.java

```diff
@@ -138,7 +138,8 @@ public QueryPublisher createQueryPublisher(
 
     if (statement.getStatement().isPullQuery()) {
       final ImmutableAnalysis analysis = ksqlEngine
-          .analyzeQueryWithNoOutputTopic(statement.getStatement(), statement.getStatementText());
+          .analyzeQueryWithNoOutputTopic(
+              statement.getStatement(), statement.getStatementText(), properties);
```

Comment:
nit: let's be consistent. If we're going to use `properties`, let's also use it below (in this file). If we're going to use `statement.getSessionConfig().getOverrides()` below, then that's use that here too. (I know they're the same, but it's unintuitive as a reader that that's the case; consistency removes that extra question.)